src/esp/gfx/WindowlessContext.cpp

```diff
@@ -4,9 +4,11 @@
 
 #include "WindowlessContext.h"
 
-#ifdef __APPLE__
+#include <Corrade/configure.h>
+
+#ifdef CORRADE_TARGET_APPLE
```

Comment:
```suggestion
#if defined(CORRADE_TARGET_APPLE)
```
?

Reply:
Sure, if you prefer that :) In my case I always pick the shortest variant.

Updated (and rebased on master as well).